packages/synapse-react-client/mocks/faker/generateFakeEntity.ts

```diff
@@ -0,0 +1,84 @@
+import { faker } from '@faker-js/faker'
+import {
+  Entity,
+  FileEntity,
+  Folder,
+  Project,
+} from '@sage-bionetworks/synapse-types'
+import { mockFileHandle } from '../mock_file_handle'
+import { pickRandomMockUser } from './fakerUtils'
+
+// When automatically generating entities, only create IDs in this range
+const MOCK_GENERATED_ENTITY_ID_SPACE_MIN = 10000
+const MOCK_GENERATED_ENTITY_ID_SPACE_MAX = 99999
```

Comment:
I figure one of the easiest ways to ensure we can get consistent generated data is to partition entity type based on the ID space. So the first `n` IDs are projects, then folders, then files, and so on.

Not that we need this much fake data today (maybe if we ever want to run a performance test).